ajax/libs/pizzicato/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "pizzicato",
+  "filename": "Pizzicato.min.js",
+  "description": "A web-audio library to simplify using and manipulating sounds.",
+  "version": "0.4.0",
+  "license": "MIT",
+  "homepage": "https://alemangui.github.io/pizzicato/",
+  "author": {
+    "name": "Alejandro Mantecon Guillen",
+    "email": "alemangui@gmail.com",
+    "url": "https://alemangui.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/alemangui/pizzicato.git"
+  },
+  "namespace": "Pizzicato",
+  "npmName": "pizzicato",
```

Comment:
I found that there are only two releases on npm but 9 on GitHub, would you like to use git auto-update so that we can grab all the 9 versions? Thanks.